test/GroupByTest.cpp

```diff
@@ -642,29 +759,41 @@ TEST_F(GroupBySpecialCount, computeGroupByForFullIndexScan) {
     Id idOfX;
     Id idOfY;
     Id idOfZ;
+    Id idOfA;
+    Id idOfB;
+    Id idOfC;
     qec->getIndex().getId("<x>", &idOfX);
     qec->getIndex().getId("<y>", &idOfY);
     qec->getIndex().getId("<z>", &idOfZ);
+    qec->getIndex().getId("<a>", &idOfA);
+    qec->getIndex().getId("<b>", &idOfB);
+    qec->getIndex().getId("<c>", &idOfC);
 
-    // Three distinct subjects.
-    ASSERT_EQ(result.size(), 3);
+    // Six distinct subjects.
+    ASSERT_EQ(result.size(), 6);
     if (includeCount) {
       ASSERT_EQ(result.numColumns(), 2);
     } else {
       ASSERT_EQ(result.numColumns(), 1);
     }
     // The test index currently consists of 6 triples.
-    EXPECT_EQ(result(0, 0), idOfX);
-    EXPECT_EQ(result(1, 0), idOfY);
-    EXPECT_EQ(result(2, 0), idOfZ);
+    EXPECT_EQ(result(0, 0), idOfA);
+    EXPECT_EQ(result(1, 0), idOfB);
+    EXPECT_EQ(result(2, 0), idOfC);
+    EXPECT_EQ(result(3, 0), idOfX);
+    EXPECT_EQ(result(4, 0), idOfY);
+    EXPECT_EQ(result(5, 0), idOfZ);
 
     if (includeCount) {
-      EXPECT_EQ(result(0, 1), Id::makeFromInt(5));
-      EXPECT_EQ(result(1, 1), Id::makeFromInt(1));
+      EXPECT_EQ(result(0, 1), Id::makeFromInt(2));
+      EXPECT_EQ(result(1, 1), Id::makeFromInt(2));
+      EXPECT_EQ(result(2, 1), Id::makeFromInt(2));
+      EXPECT_EQ(result(3, 1), Id::makeFromInt(7));
+      EXPECT_EQ(result(4, 1), Id::makeFromInt(1));
       // TODO<joka921> This should be 1.
       // There is one triple added <z> @en@<label> "zz"@en which is
       // currently not filtered out.
-      EXPECT_EQ(result(2, 1), Id::makeFromInt(2));
+      EXPECT_EQ(result(5, 1), Id::makeFromInt(2));
```

Comment:
Here you can do the exakt same thing with the `IdTestHelper` `IntId` or whatever it is called
(have a look at our test helpers in general).
Then also this whole test becomes a single line with the `getColumn(...)` as above.